### ISSUE TITLE
Questions about the Data Set 

### ISSUE THREAD

**Issue by ghost** [msnews/msnews.github.io#18]
As described in the paper,  There are 2,186,683 samples in the training set, 365,200 samples in the validation set, and 2,341,619 samples in the test set.
But after I downloaded the dataset and analyzed it, I noticed that there are  223,274,8 samples in the training set,  376,471 samples in the validation set, and 237,072,7 samples in the test set.

I'm wondering whether there were some changes in the dataset?
Thank you so much.
